java/test/jmri/util/JUnitUtil.java

```diff
@@ -199,10 +199,13 @@ public class JUnitUtil {
     static private String initPrefsDir = null;
 
     /**
-     * JMRI standard setUp for tests that mock the InstanceManager. This should be the first line in the {@code @Before}
+     * JMRI standard setUp for tests that mock the InstanceManager.
+     * This should be the first line in the {@code @BeforeEach}
      * annotated method if the tests mock the InstanceManager.
+     * <p>
      * One or the other of {@link #setUp()} or {@link #setUpLoggingAndCommonProperties()} must
-     * be present in the {@code @Before} routine.
+     * be present in the {@code @BeforeEach} routine.
+     * <p>
      */
     public static void setUpLoggingAndCommonProperties() {
```

Comment:
@bobjacobsen @icklesteve 
This JavaDoc doesn't seem to be correct. The change @icklesteve did is OK, but the JavaDoc as it was before doesn't look right. I suppose this method should never be called by JUnit tests, only by JUnitUtil. Copy/paste error?